src/devices/Bh1745/Bh1745.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <Compile Include="*.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)/../Common/System/Runtime/CompilerServices/IsExternalInit.cs"/>
```

Comment:
```suggestion
    <Compile Include="$(MSBuildThisFileDirectory)/../Common/System/Runtime/CompilerServices/IsExternalInit.cs">
      <Link>Common/System/Runtime/CompilerServices/IsExternalInit.cs</Link>
    </Compile>
```
Nit: Add a Link here for VS Experience.

Reply:
If you switch the order of the TFMs in `Bh1745.csproj` or load these projects on a machine within .NET 5.0, you will get the following experience, with `IsExternalInit.cs` listed. Just ignore it.

![image](https://user-images.githubusercontent.com/2608468/98614966-70612280-22ae-11eb-928e-492140cb01cc.png)